src/python/T0/ConditionUpload/upload.py

```diff
@@ -555,7 +555,13 @@ def uploadFile(self, filename, backend = defaultBackend, temporaryFile = default
 
         with tempfile.NamedTemporaryFile() as metadata:
             with open('%s.txt' % basepath, 'rb') as originalMetadata:
-                json.dump(json.load(originalMetadata), metadata, sort_keys = True, indent = 4)
+                originalMetadata_dic = json.load(originalMetadata)
+                originalMetadata_encode = json.dumps(originalMetadata_dic, indent = 2).encode('utf-8')
+                try:
+                    json.dump(originalMetadata_encode, metadata, sort_keys = True, indent = 4)
+                except TypeError as err:
+                    msg = 'raised a %s when ' % err.__repr__()
```

Comment:
Is this error message incomplete?

Reply:
Sorry German, that was a typo. Is fixed now